tests/unit/badge/badge.renderBadge.spec.js

```diff
@@ -0,0 +1,42 @@
+import { shallow } from 'enzyme';
```

Comment:
better to have small files, because:

1. all test in one file related to one function (single responsibility principle + separation of concerns)
2. in this case we have separate file with `snapshots`
3. reduce conflicts
4. easy to find failed test
5. same approach like we have on core project(rwd)
from my point of view we have only pros without cons ;)